packages/filesystem/src/browser/download/file-download-service.ts

```diff
@@ -147,54 +142,68 @@ export class FileDownloadService {
         return this.deferredUpload.promise;
     }
 
+    protected handleCopy(event: ClipboardEvent, downloadUrl: string) {
```

Comment:
All methods and functions with `protected` and `public` (default) visibility, must declare the return type. For instance, here, it is `: void`.

Please, apply this rule to all the methods you have introduced.